### PR TITLE
(Fixes #32) Support additional security group

### LINKED ISSUE #32: Add security group ID support for EMR

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

**Comment by realstraw** [comment]
@schowsf can you review pls

File: README.md [hunk 1]
```diff
@@ -63,101 +63,23 @@ Orchard allows the definition and execution of **workflows**, where each workflo
 
 Below is an example workflow that defines a number of activities to be executed in an AWS VPC environment:
 
+You can generate a an example workflow with the following command:
+
+```
+cd example/data/data
```
Comment (schowsf) [inline README.md:69]: I tried with the following for mustache to work
```
cd example/data
```

File: README.md [hunk 2]
```diff
@@ -63,101 +63,23 @@ Orchard allows the definition and execution of **workflows**, where each workflo
 
 Below is an example workflow that defines a number of activities to be executed in an AWS VPC environment:
 
+You can generate a an example workflow with the following command:
```
Comment (schowsf) [inline README.md:66]: typo^^ (no a)

generate an example.

File: README.md (hunk 1 above)
Comment (realstraw) [reply README.md]: thanks, it's fixed

File: README.md (hunk 2 above)
Comment (realstraw) [reply README.md]: thanks, it's fixed

**Review by schowsf — APPROVED** [review]
LGTM

**Pull request merged**